packages/node/src/cli.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+/* eslint-disable no-eval */
+
+const fs = require('fs')
+const yargs = require('yargs')
+const yargsFlags = require('./yargsFlags')
+const { runLava } = require('./index')
+
+const defaults = require('./defaults')
+
+runLava(parseArgs()).catch(err => {
+  // explicity log stack to workaround https://github.com/endojs/endo/issues/944
+  console.error(err.stack || err)
+  process.exit(1)
+})
+
+
+function parseArgs () {
+  const argsParser = yargs
+    .usage('$0 <entryPath>', 'start the application', (yargs) => {
+      // the entry file to run (or parse)
+      yargs.positional('entryPath', {
+        describe: 'the path to the entry file for your application. same as node.js',
+        type: 'string'
+      })
+      yargsFlags(yargs, defaults)
```

Comment:
I could unwrap yargsFlags back into this place now

Reply:
I think that this makes sense, as it would reduce the cognitive complexity (not that it is making it much harder... but one less thing for our brains to worry about 😅)

Reply:
Nope, turns out I need that shared. There's a command runner in lavamoat that uses it too. Updating it now.